config/appConfig.json

```diff
@@ -1,23 +1,25 @@
 {
     "appConfig": {
         "tweetsSearchBaseUrl": "http://54.158.101.33:8080/bopws/tweets/search?",
-        "tweetsExportBaseUrl": "http://54.158.101.33:8080/bopws/tweets/export?"
+        "tweetsExportBaseUrl": "http://54.158.101.33:8080/bopws/tweets/export?",
+        "ratioInnerBbox": 0.9
     },
     "bopwsConfig": {
-        "heatmapFacetLimit": 1000,
+        "heatmapFacetLimit": 10000,
         "csvDocsLimit": 1
     },
     "mapConfig": {
         "renderer": "canvas",
         "view": {
-            "center": [430493, 6887893],
-            "maxZoom": 10,
-            "minZoom": 2,
+            "center": [0, 0],
+            "extent": [-20000000.00,-20000000.00,20000000.00,20000000.00],
```

Comment:
This extent is especially used for the initial extent of the query bounding box. I changed this to official mercator bounds in 5c46a22